package.json

```diff
@@ -63,6 +63,7 @@
     "angular-ui-router": "^0.4.2",
     "lodash": "^4.17.4",
     "ovh-documentation-toolkit": "https://github.com/ovh-ux/ovh-documentation-toolkit.git",
+    "ovh-ui-angular": "https://github.com/ovh-ux/ovh-ui-angular.git#develop",
```

Comment:
Can we do an alpha version of ovh-ui-angular ? Just to be sure that nobody break the build without knowing ?